hack/common.sh

```diff
@@ -33,6 +33,93 @@ get_OCP_version() {
   echo $OCP_VER_MAJOR.$OCP_VER_MINOR
 }
 
+check_git_branch(){
```

Comment:
Please add a space after `()`. Applies to all the function where the space is not present.

Please add a comment for the function.